rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/exceptions/AppServicesException.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.service.bridge.rhoas.exceptions;
+
+public class AppServicesException extends RuntimeException {
```

Comment:
@kostola Same here. Should this extend the `EventBridgeManagerException`?

Reply:
@robpblake I can't find the exception you mentioned. It seems like it was removed in some refactoring. Am I wrong?

Reply:
@kostola You're right. It was refactored as part of #183 by @r00ta . `InternalPlatformException` is the one to use now 👍

Reply:
Done :+1: